### PR TITLE
sort dynos before printing

### PR DESCRIPTION
**Pull request by RasPhilCo**
Closes https://github.com/heroku/cli/issues/615

### CODE REVIEW

File: src/commands/ps/index.js [hunk 1]
```diff
@@ -163,10 +164,10 @@ Examples:
     $ heroku ps
     === run: one-off dyno
     run.1: up for 5m: bash
-
```
Comment (jdxcode) [inline src/commands/ps/index.js]: This actually needs to be there for the markdown formatting

Comment (RasPhilCo) [reply src/commands/ps/index.js]: Gotcha. My linter did that automatically, so I'll be mindful about that going forward.

Comment (jdxcode) [reply src/commands/ps/index.js]: yeah this is one reason why I'd like to make examples more formal, so we don't have to have the entire markdown format in the string

**Comment by RasPhilCo** [comment]
@jdxcode gtg

**Comment by jdxcode** [comment]
this doesn't appear to work, I added a test that I think should be passing

**Comment by RasPhilCo** [comment]
@jdxcode good call on test, just had to sort the expected output aswell.

**Pull request merged**